website/settings/defaults.py

```diff
@@ -56,6 +56,9 @@ def parent_dir(path):
 ELASTIC_TIMEOUT = 10
 ELASTIC_INDEX = 'website'
 SHARE_ELASTIC_URI = ELASTIC_URI
+SHARE_VERSION = 1
+SHARE_ELASTIC_INDEX_TEMPLATE = 'share_v{}'
+SHARE_ELASTIC_INDEX = SHARE_ELASTIC_INDEX_TEMPLATE.format(SHARE_VERSION)
```

Comment:
The actual SHARE_ELASTIC_INDEX should default to ```share```. ```share``` is an index that is aliased to whatever the latest version of the metadata schema is. ```share_vx``` is used for accessing older versions of the schema.